src/App.js

```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
 import './App.css';
+import Calculator from './Components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <main>
+      <section>
+        <Calculator />
+      </section>
+    </main>
```

Comment:
Please make sure your `App.js` file uses a class-based component as required 👍🏽

Reply:
Hi @rotimiazeez, thank you for your observation. I thought that classes were needed only for the component. But now I have also included them in the app.js file.